bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -311,6 +311,31 @@ const BaseContainer = withTracker(() => {
     },
   });
 
+  Users.find({}, { fields: { validated: 1, name: 1 } }).observe({
```

Comment:
Please consider adding `if (Settings.application.userJoinAudioAlerts || Settings.application.userJoinPushAlerts)` so we don't observe unless we need to

Reply:
I would also consider alerting only moderators. I don't see value in notifying viewers about users joining

Reply:
Technically I think it's possible for this code to notify me for my arrival. Could filter by userId as well

Reply:
The observer will now only exist if one of the settings is turned on. I also added an exclusion to not alert if the changed user is the local user.